assets/stylesheets/layout/_sidebar.scss

```diff
@@ -126,9 +126,7 @@
 		overflow: hidden;
 		display: flex;
 
-		&:focus {
-			outline: none;
-		}
+		-webkit-tap-highlight-color: rgba($gray-dark, .2);
```

Comment:
I'm not a designer but in the event that this is otherwise good to go, it needs to be spaced out like `rgba( $gray-dark, .2 )` per our [CSS guidelines](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#spacing).